api/idea-card/config/routes.json

```diff
@@ -25,9 +25,9 @@
       }
     },
     {
-      "method": "GET",
-      "path": "/users/:id/idea-cards",
-      "handler": "idea-card.findByUserId",
+      "method": "POST",
+      "path": "/idea-cards/:id/comment",
```

Comment:
Does this mean idea cards are not tied to users anymore?

Reply:
For this iteration, they are not.